snuba/clickhouse/query.py

```diff
@@ -7,43 +7,42 @@
 
 
 class ClickhouseQuery:
```

Comment:
It's not any different/worse than it was before, but the `__repr__` for this class won't be very useful (it won't tell you anything about the type of query being represented here.)

Reply:
Will file a ticket for this, I think a good __repr__ should rely on Query.__repr__ that has the same problem.